includes/controls/alert.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor alert control.
+ *
+ * A base control for creating alerts in the Editor panels.
+ *
+ * @since 3.20.0
+ */
+class Control_Alert extends Base_UI_Control {
+
+	/**
+	 * Get alert control type.
+	 *
+	 * Retrieve the control type, in this case `alert`.
+	 *
+	 * @since 3.20.0
+	 * @access public
+	 *
+	 * @return string Control type.
+	 */
+	public function get_type() {
+		return 'alert';
+	}
+
+	/**
+	 * Render alert control output in the editor.
+	 *
+	 * Used to generate the control HTML in the editor using Underscore JS
+	 * template. The variables for the class are available using `data` JS
+	 * object.
+	 *
+	 * @since 3.20.0
+	 * @access public
+	 */
+	public function content_template() {
+		?>
+		<# data.content = elementor.compileTemplate( data.content, { view } ); #>
+		<div class="elementor-control-alert elementor-panel-alert elementor-panel-alert-{{ data.alert_type }}">
+			<# if ( data.heading ) { #>
+			<div class="elementor-control-alert-heading">{{{ data.heading }}}</div>
+			<# } #>
+			<# if ( data.content ) { #>
+			<div class="elementor-control-alert-content ">{{{ data.content }}}</div>
+			<# } #>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Get alert control default settings.
+	 *
+	 * Retrieve the default settings of the alert control. Used to return the
+	 * default settings while initializing the alert control.
+	 *
+	 * @since 3.20.0
+	 * @access protected
+	 *
+	 * @return array Control default settings.
+	 */
+	protected function get_default_settings() {
+		return [
+			'alert_type' => 'info', // info, success, warning, danger.
+			'heading' => '',
+			'content' => '',
+		];
+	}
```

Comment:
I would suggest doing the following things:

set consts for `info`, `success`, `warning`, `danger`, see comment above. 

```suggestion
         * @param string $alert_type see alert_type_* constants
	 * @since 3.20.0
	 * @access protected
	 *
	 * @return array Control default settings.
	 */
	protected function get_default_settings( $alert_type ) {
		return [
			'alert_type' => $alert_type
			'heading' => '',
			'content' => '',
		];
	}
```

Then the function call will be:
`$this->get_default_settings( Control_Alert::alert_type_info )`.